starter_code/elevator.js

```diff
@@ -1,19 +1,88 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
-  }
-
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+  constructor() {
+    this.floor = 0;
+    this.MAXFLOOR = 10;
+    this.direction = 'up';
+    this.interval;
+    this.requests = [];
+    this.waitingList = [];
+    this.passengers = [];
+  }
+
+  start() {
+    this.interval = setInterval(() => {
+      this.requests.forEach((request) => {
+        this._passengersEnter();
+        this._passengersLeave();
+        if (request > this.floor) {
```

Comment:
Este `if` era candidato TOP para un ternario!!!
`request > this.floor ? this.floorUp() : this.floorDown()`
Mira qué maravilla!!! :D